packages/docs-theme/src/components/example.tsx

```diff
@@ -121,16 +133,15 @@ export class Example extends React.PureComponent<IDocsExampleProps> {
     }
 
     public componentDidMount() {
-        if (this.props.forceUpdate === false) {
-            return;
+        // `forceUpdate` - The docs app suffers from a Flash of Unstyled Content
+        // that causes components to mis-measure themselves on first render.
+        // Delay initial render till the DOM loads with a requestAnimationFrame.
+        const { forceUpdate = true } = this.props;
```

Comment:
prefer having defaults in one place (`defaultProps` since you're using it twice)